docs/ext/pydantic_autosummary/generate.py

```diff
@@ -0,0 +1,880 @@
+            imported or getattr(value, "__module__", None) == obj.__name__
+        ):
+            skipped = _skip_member(app, value, name, documenter.objtype)
+            if skipped is True:
+                pass
+            elif skipped is False:
+                # show the member forcedly
+                items.append(name)
+                public.append(name)
+            else:
+                items.append(name)
+                if name in include_public or not name.startswith("_"):
+                    # considers member as public
+                    public.append(name)
+    return public, items
+
+
+def _get_module_attrs(name: str, members: Any) -> tuple[list[str], list[str]]:
+    """Find module attributes with docstrings."""
+    attrs, public = [], []
+    try:
+        analyzer = ModuleAnalyzer.for_module(name)
+        attr_docs = analyzer.find_attr_docs()
+        for namespace, attr_name in attr_docs:
+            if namespace == "" and attr_name in members:
+                attrs.append(attr_name)
+                if not attr_name.startswith("_"):
+                    public.append(attr_name)
+    except PycodeError:
+        pass  # give up if ModuleAnalyzer fails to parse code
+    return public, attrs
+
+
+def _get_modules(
+    obj: Any,
+    *,
+    skip: Sequence[str],
+    name: str,
+    public_members: Sequence[str] | None = None,
+) -> tuple[list[str], list[str]]:
+    items: list[str] = []
+    public: list[str] = []
+    for _, modname, _ispkg in pkgutil.iter_modules(obj.__path__):
+        if modname in skip:
+            # module was overwritten in __init__.py, so not accessible
+            continue
+        fullname = name + "." + modname
+        try:
+            module = import_module(fullname)
+            if module and hasattr(module, "__sphinx_mock__"):
+                continue
+        except ImportError:
+            pass
+
+        items.append(fullname)
+        if public_members is not None:
+            if modname in public_members:
+                public.append(fullname)
+        else:
+            if not modname.startswith("_"):
+                public.append(fullname)
+    return public, items
+
+
+def generate_autosummary_docs(
+    sources: list[str],
+    output_dir: str | os.PathLike[str] | None = None,
+    suffix: str = ".rst",
+    base_path: str | os.PathLike[str] | None = None,
+    imported_members: bool = False,
+    app: Any = None,
+    overwrite: bool = True,
+    encoding: str = "utf-8",
+) -> None:
+    showed_sources = sorted(sources)
+    if len(showed_sources) > 20:
+        showed_sources = showed_sources[:10] + ["..."] + showed_sources[-10:]
+
+    # ----------- pydantic_autosummary change
+    logger.info(
+        __("[pydantic_autosummary] generating autosummary for: %s")
+        % ", ".join(showed_sources)
+    )
+    if output_dir:
+        logger.info(__("[pydantic_autosummary] writing to %s") % output_dir)
+    # ----------/ pydantic_autosummary change
```

Comment:
Just changed the logger to use `pydantic_autosummary: ` instead of `autosummary: `